public/index.html

```diff
@@ -7,5 +7,388 @@
+                <span class="product__price">$859</span>
+                <span class="product__price--discount">$899</span>
+              </div>
+              <div class="product__characteristics">
+                <span class="product__spec">Screen</span>
+                <span class="product__spec--info">5.8” OLED</span>
+              </div>
+              <div class="product__characteristics">
+                <span class="product__spec">Capacity</span>
+                <span class="product__spec--info">256 GB</span>
+              </div>
+              <div class="product__characteristics">
+                <span class="product__spec">RAM</span>
+                <span class="product__spec--info">3 GB</span>
+              </div>
+              <div class="product__buttons">
+                <button class="product__buy button">Add to cart</button>
+                <div class="product__fav"></div>
+              </div>
+            </article>
+          </div>
+        </section>
+
+        <section class="categories section">
+          <h2 class="section__heading">Shop by category</h2>
+          <div class="categories__catalog">
+            <article class="categories__category">
+              <a href="#" class="categories__image-container categories__image-container--phones">
+                <img
+                  src="./img/phones.svg"
+                  class="categories__image"
+                  alt="Mobile phones"
+                >
+              </a>
+              <a href="#" class="categories__name">Mobile phones</a>
+              <p class="categories__quantity">95 models</p>
+            </article>
+
+            <article class="categories__category">
+              <a href="#" class="categories__image-container categories__image-container--tablets">
+                <img
+                  src="./img/tablets.svg"
+                  class="categories__image"
+                  alt="Mobile phones"
+                >
+              </a>
+              <a href="#" class="categories__name">Tablets</a>
+              <p class="categories__quantity">24 models</p>
+            </article>
+
+            <article class="categories__category">
+              <a href="#" class="categories__image-container categories__image-container--accessories">
+                <img
+                  src="./img/accessories.svg"
+                  class="categories__image"
+                  alt="Mobile phones"
+                >
+              </a>
+              <a href="#" class="categories__name">Accessories</a>
+              <p class="categories__quantity">100 models</p>
+            </article>
+          </div>
+        </section>
+
+        <section class="new section">
+          <div class="section__top">
+            <h2 class="section__heading">Brand new models</h2>
+            <div class="scroll-container">
+              <div class="scroll scroll--left"></div>
+              <div class="scroll scroll--right"></div>
```

Comment:
there should be icons

Reply:
We've discussed this question today. I'll leave it as is. For now. If you don't mind